packages/site/src/globalTypes.d.ts

```diff
@@ -0,0 +1,7 @@
+interface AtlantisWindow extends Window {
```

Comment:
Is there a better way to do this in 2024? This is how I did it forever ago and GPT also gave me this answer but it always felt wrong, like there should be a generic or something we can use instead.